tests/js/server/shell/shell-general-graph-creation-cluster.js

```diff
@@ -56,6 +56,8 @@ function GeneralGraphClusterCreationSuite() {
         graph._drop(gn, true);
       } catch (ignore) {
       }
+      db._drop(vn);
```

Comment:
Why is that necessary? `vn` and `en` should be dropped automatically with `graph._drop(gn, true);`

Reply:
I got a failure once that the two collections (vertex & edge) were not cleaned up after the test. So I added the drop commands for extra safety.
Shouldn't do any harm